libs/net/dal/Configuration/TopicScoreRuleConfiguration.cs

```diff
@@ -11,8 +11,8 @@ public override void Configure(EntityTypeBuilder<TopicScoreRule> builder)
         builder.Property(m => m.SourceId).IsRequired();
         builder.Property(m => m.SeriesId);
         builder.Property(m => m.Section).HasMaxLength(100);
-        builder.Property(m => m.PageMin);
-        builder.Property(m => m.PageMax);
+        builder.Property(m => m.PageMin).HasMaxLength(5);
```

Comment:
Change to string value.  I don't know how this wasn't included in my prior PR.  I think there was a git merge issue I must have run into because I did convert this to a string at one point.